config/seata-config-core/src/main/java/io/seata/config/ConfigurationFactory.java

```diff
@@ -55,9 +60,11 @@
         Configuration extConfiguration = null;
         try {
             extConfiguration = EnhancedServiceLoader.load(ExtConfigurationProvider.class).provide(configuration);
-
-        } catch (Exception ignore) {
-
+            if (LOGGER.isInfoEnabled()) {
+                LOGGER.info("load extConfiguration:{}", extConfiguration.getClass().getSimpleName());
+            }
+        } catch (Exception e) {
+            LOGGER.warn("failed to load extConfiguration:{}", e);
```

Comment:
`LOGGER.warn("failed to load extConfiguration:{}", e.getMessag(), e);`

Reply:
@objcoding please correct the log format.

Reply:
> @objcoding please correct the log format.

ok.